src/controllers/utils/index.js

```diff
@@ -19,6 +25,20 @@ const constructRegexQuery = ({ isUsingMainKey, searchWord }) => (
       : /^[.{0,}\n{0,}]/
 );
 
+/* Determines that the parameter gotten as a wordClass when
+ * searching for words from a given wordClass is actually valid
+ */
+export const parseWordClass = (param) => {
+  if (!param) {
+    throw new Error('No wordClass specified');
+  }
+  const valueOfWordClass = findKey(wordClass, { value: param });
+  if (valueOfWordClass === undefined) {
```

Comment:
These two lines can be rewritten like:

```js
const wordClassValue = wordClass[param];
if (!wordClassValue) {
```

Two things are happening here:
1. You don't need to import `findKey` since the key that you are trying to get is the same as the `value`. So you can just do `wordClass[param]`
2. `!value` is more succinct that `value === undefined`